pkg/ebpf/c/tc.v4egress.bpf.c

```diff
@@ -61,6 +61,8 @@ struct data_t {
 	__u32  dest_port;
 	__u32  protocol;
 	__u32  verdict;
+	__u32 packet_sz;
+	__u8 is_egress;
```

Comment:
Why do we use unsigned 8 here? Is it this a reference from kernel data structure or since the value be 0 or 1 we chose the `__u8` ?

Reply:
Its just to store boolean value